pkg/ddevapp/config.go

```diff
@@ -939,6 +939,9 @@ func (app *DdevApp) RenderComposeYAML() (string, error) {
 		extraWebContent = extraWebContent + "\nRUN npm install -g n"
 		extraWebContent = extraWebContent + fmt.Sprintf("\nRUN n install %s && ln -sf /usr/local/bin/node /usr/local/bin/nodejs", app.NodeJSVersion)
 	}
+	if app.CorepackEnable && app.NodeJSVersion >= "18" {
```

Comment:
```suggestion
	if app.CorepackEnable {
```

It's probably better to let the user decide what to do. This check would have unexpected results with `nodejs_version` `20.1.0`, `v20`, etc.